### ISSUE TITLE
no war files for v1.2020.7 - v1.2020.9

### ISSUE THREAD

**Issue by anb0s** [plantuml/plantuml-server#144]
There are no war files for the new versions v1.2020.7 and v1.2020.8 under [release](https://github.com/plantuml/plantuml-server/releases) assets.

**Comment by anb0s** [comment]
found at sourceforge:
https://sourceforge.net/projects/plantuml/files/1.2020.7/
https://sourceforge.net/projects/plantuml/files/1.2020.8/

**Comment by arnaudroques** [comment]
@jarv This may be caused by this last commit https://github.com/plantuml/plantuml-server/commit/51e0922eeaecb233670a7d50d4f9d96be9dd5cfa
Any idea ?

**Comment by jarv** [comment]
@arnaudroques there were some invalid `tag:` lines and it looks like I accidentally removed the one under `build:`, I believe https://github.com/plantuml/plantuml-server/pull/145 should resolve it.

**Comment by arnaudroques** [comment]
Ok, great.
I've just merge your contribution.
Thanks for your help !

**Comment by jarv** [comment]
https://github.com/plantuml/plantuml-server/releases/tag/v1.2020.9 is still missing the war file as a release assets 

From the log:

```
Preparing deploy

Cleaning up git repository with `git stash --all`. If you need build artifacts for deployment, set `deploy.skip_cleanup: true`. See https://docs.travis-ci.com/user/deployment#Uploading-Files-and-skip_cleanup.

Saved working directory and index state WIP on (no branch): f355dc6 version 1.2020.9

target/[secure]-v1.2020.9.war does not exist.

Setting target_commitish to f355dc6666cd6a3339f3c542a6c1ab94da1a23ee

dpl.3

Deploying application

Already up to date!

Not currently on any branch.

nothing to commit, working tree clean

Dropped refs/stash@{0} (e3d1d6396919638c60858caf06bf63fdad58a21c)

Done. Your build exited with 0.

```

Resolved with https://github.com/plantuml/plantuml-server/pull/148

This should hopefully be the last change for this to work :)

**Comment by arnaudroques** [comment]
"The more it fails, the more likely it is that it will work" :-)
Thanks for your help !

**Comment by anb0s** [comment]
fixed with v1.2020.10, Thanks 👍
